.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 bower_components/
 npm-debug.log
+.DS_STORE
```

Comment:
Please use a global `.gitignore` to ignore computer/text editor specific files like that.
The config `core.excludesfile` can be use to define a `~/.gitignore`.
see https://git-scm.com/docs/gitignore/1.7.12